src/main/java/org/prebid/server/auction/BidResponseCreator.java

```diff
@@ -636,7 +640,7 @@ private Bid toBid(BidderBid bidderBid, String bidder, ExtRequestTargeting target
             final Map<BidType, TargetingKeywordsCreator> keywordsCreatorByBidType =
                     keywordsCreatorByBidType(targeting, isApp);
             final boolean isWinningBid = winningBids.contains(bid);
-            final String winUrl = eventsEnabled && bidType != BidType.video
+            final String winUrl = eventsEnabled && eventsAllowedByRequest && bidType != BidType.video
```

Comment:
According to requirements `eventsAllowedByRequest` should not be applied here.
i mean it should not affect the `hb_winurl` & `hb_bidid` targeting keywords.